jax/_src/prng.py

```diff
@@ -169,6 +169,8 @@ def ndim(self):
   def dtype(self):
     return KeyTy(self.impl)
 
+  _device = property(op.attrgetter('_base_array._device'))
```

Comment:
Drive-by: For jax.Array you will also need to add `_committed` attributed for it to pass on the Array path.

Reply:
Thanks!